sites/fast-website/src/app/css/style.css

```diff
@@ -186,21 +186,46 @@ fast-divider::after {
   margin: 20px 0;
 }
 
-.section-badge {
-  color: var(--accent-fill-rest);
+fast-badge {
+  --badge-fill-primary: var(--neutral-fill-focus);
+  --badge-color-accent: var(--accent-foreground-rest);
```

Comment:
are these meant to be used together? Could we just have them both be "primary"?

Reply:
I am a bit confused by your question. One sets the color and one sets the fill. If they are both set to primary wouldn't the color and fill then be the same color meaning that you could not see the text?

Reply:
Oh I see what you are saying changing the color variable to be badge-color-primary instead of badge-color-accent. Yes, I will change that. Thank you.